src/Components/FollowButton/FollowArtistButton.tsx

```diff
@@ -56,31 +57,42 @@ export class FollowArtistButton extends React.Component<Props> {
   handleFollow = () => {
     const { artist, currentUser, relay, onOpenAuthModal } = this.props
 
+    const newFollowCount = artist.is_followed
```

Comment:
If you currently follow the artist, that means you have pressed the button and will now unfollow, so the new count is decremented.